contracts/DADBridgeWrapper.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.8.9;
+
+import { AccessControl } from "@openzeppelin/contracts/access/AccessControl.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract DADBridgeWrapper is AccessControl {
+    using SafeERC20 for IERC20;
+
+    bytes32 public constant ROLE_ADMIN = keccak256("ROLE_ADMIN");
+
+    IERC20 public immutable bntToken;
```

Comment:
there's going to be a different instance per token

Reply:
Not sure what you mean.
We have two token pointers here - `bntToken` and `ethBntToken`; what other token is needed?

Reply:
there should just be a single token - the contract should be generic and for the 2 bridges (ETHBNT/BNT), there will be 2 instances of this contract

Reply:
Really? How would you make sure that they execute the exact same transaction (recipient and amount) on each contract?

Reply:
that's for the bridge to take care of.
and in any case, it doesn't make sense that when someone sends 10 BNT from EOS to Ethereum, they get both 10 BNT and 10 ETHBNT on the ethereum side

Reply:
So just a single ERC20 token pointer?

Reply:
yup, simple

Reply:
Done.